recipes/ngscheckmate/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd $PREFIX
+mv $SRC_DIR $PREFIX/NGSCheckMate-1.0.0
+
+mkdir -p $PREFIX/bin
+cat << EOF > $PREFIX/bin/ncm.py
+#!/usr/bin/env bash
+
+python /usr/local/NGSCheckMate-1.0.0/ncm.py "\$@"
```

Comment:
We'll need to think about this, because on the conda installs `ncm.py` will not go to `/usr/local/`

Reply:
Thank you for this feedback. The main issue I'm having currently is for some of my recent commits the DevOps tests have gone beyond the walltime of 6 hours and have subsequently failed (e.g. see commit 435daae). I believe it's linked to the requirements I included in the meta.yaml file; the problem goes away if I don't include R in the requirements. Do you know why this is happening?

Reply:
Can the Python and Samtools version requirements be loosened a bit?

Reply:
Hi, thanks for looking at this. I made the version requirements as loose as possible (python needs to be 2.7*), still looks to be having issues. The bit it seems to get stuck on is while collecting package metadata (repodata.json).

Reply:
I played around, and yeah it takes forever to solve the environment. I used Mamba and it solved it pretty quickly. I've pinned the versions that mamba picked, then reran with conda and it solved it fine.

Are the versions ok? 
```
python 2.7.18
R 4.1
samtools 1.14
```

Reply:
I've given it a test and those versions are good. Thanks a lot, I'm going to have a think about how to handle ncm.py